lib/features/recommendation/organisations/widgets/organisation_detail_bottomsheet.dart

```diff
@@ -103,15 +105,21 @@ class OrganisationDetailBottomSheet extends StatelessWidget {
           ),
           floatingActionButtonLocation:
               FloatingActionButtonLocation.centerDocked,
-          floatingActionButton: GivtFloatingActionButton(
+          floatingActionButton: GivtElevatedButton(
             text: "Donate",
-            onPressed: () {
+            isDisabled: false,
+            onTap: () {
               AnalyticsHelper.logEvent(
                 eventName: AmplitudeEvent.donateToRecommendedCharityPressed,
                 eventProperties: {
                   AnalyticsHelper.charityNameKey: organisation.name,
                 },
               );
+              if (context.read<FlowsCubit>().state.flowType !=
+                  FlowType.recommendation) {
+                context.read<FlowsCubit>().startRecommendationFlow();
```

Comment:
why do we call startRecommendationFlow() here? I don't think that is the right place

Reply:
There was a bug that would take me from orgDetailsBottomsheet to coinFlow, but I cannot recreate it, so I removed this logic. It should be in a separate PR anw